Cheat sheet instructions for ansible ad-hoc commands (#1171)
ansible/ansible-documentation: docs/docsite/rst/command_guide/cheatsheet.rst

  - extracting example.role to /home/user/.ansible/roles/example.role
  - example.role was installed successfully

* List all installed roles:

.. code-block:: bash

  ansible-galaxy role list

See :ref:`ansible-galaxy` for detailed documentation.

ansible
=======

Running ad-hoc commands
^^^^^^^^^^^^^^^^^^^^^^^

* Install a package

.. code-block:: bash

  ansible localhost -m ansible.builtin.apt -a "name=apache2 state=present" -b -K

> Runs  ``ansible localhost``- from your local system

Reviewer: ```suggestion
Runs  ``ansible localhost`` on your local system.
```

> - ``name=apache2 state=present`` - this installs the package `apache2` on a Debian based system.

Reviewer: ```suggestion
- ``name=apache2 state=present`` installs the `apache2` package on a Debian-based system.
```

> - ``-b`` - executes with elevated privileges (uses :ref:`become <become>`).

Reviewer: ```suggestion
- ``-b`` uses :ref:`become <become>` to execute with elevated privileges.
```

> - ``become`` - executes with root privileges.

Reviewer: Should we remove this line as it is redundant with the previous one?

Author: Yeah I think we can remove it.

> - ``-m`` - module name.

Reviewer: ```suggestion
- ``-m`` specifies a module name.
```

> - ``-K`` - prompts to ask for privilege escalation password (sudo password).

Reviewer: ```suggestion
- ``-K`` prompts for the privilege escalation password.
```

Reviewer: I'm not sure it's 100% always the case this is the root user password. I'd just keep it consistent with the general docs: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_privilege_escalation.html#become-command-line-options

  - extracting example.role to /home/user/.ansible/roles/example.role
  - example.role was installed successfully

* List all installed roles:

.. code-block:: bash

  ansible-galaxy role list

See :ref:`ansible-galaxy` for detailed documentation.

> ansible
> ================

Reviewer: ```suggestion
ansible
=======
```

> Running ad-hoc commands:
> ^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Running ad-hoc commands
^^^^^^^^^^^^^^^^^^^^^^^
```

  ansible-galaxy role list

See :ref:`ansible-galaxy` for detailed documentation.

ansible
=======

Running ad-hoc commands
^^^^^^^^^^^^^^^^^^^^^^^

* Copy a file

.. code-block:: bash

    ansible localhost -m ansible.builtin.copy -a "src=/etc/hosts dest=/tmp/hosts"

This copies the `/etc/hosts` file to `/tmp/hosts` on your localhost. You can replace `localhost` 
with any host that is configured in the ansible inventory.

* Install a package

.. code-block:: bash

    ansible localhost -m ansible.builtin.apt -a "name=apache2 state=present" -b -K

This installs the package `apache2` on a Debian based system. Two other parameters are shown here, one
is `-b` which instructs ansible to run the operation with `become` and `-K` will prompt to ask for 
privilege escalation password (sudo password).

* Manage a service

.. code-block:: bash

    ansible localhost -m ansible.builtin.service -a "name=apache2 state=stopped" -b -K

This stops the `apache2` service. During installatione earlier, it was automatically started. So this 
ad-hoc command stops the service.

How to identify that the service has indeed been stopped? The above ansible ad-hoc command will show output 

> like this:

Reviewer 2: ```suggestion
as follows:
```

Reviewer 2: "like" can mean different things, especially for non-native English speakers so we try to avoid it where we can.

.. code-block:: bash

    localhost | CHANGED => {
    "changed": true,
    "name": "apache2",
    "state": "stopped",

>     ...

Reviewer: ```suggestion
    # ...
```

Reviewer: It might be a good idea to comment the ellipsis here in a similar way to the YAML samples. Here is a recent PR that did the same: https://github.com/ansible/ansible-documentation/pull/1114
